src/com/projectkorra/rpg/worldevent/util/WorldEventDisplayManager.java

```diff
@@ -61,11 +38,11 @@ public BossBar createBossBar(World world, WorldEvent event) {
 				size++;
 				flags.add(BarFlag.CREATE_FOG);
 			}
-			BossBar bar = plugin.getServer().createBossBar(event.getElement().getColor() + event.getName(), getBarColor(event.getElement()), BarStyle.SOLID, flags.toArray(new BarFlag[size]));
+			BossBar bar = plugin.getServer().createBossBar("�l" + event.getTextColor() + event.getName(), event.getBarColor(), BarStyle.SOLID, flags.toArray(new BarFlag[size]));
```

Comment:
Maybe we change this to `ChatColor.BOLD` instead of a hardcoded `§l` to improve readability?

Reply:
In addition to readability it reduces the hassle of copy pasting the character (or typing the ASCII code) since its not native to most keyboard layouts. In addition the code should mimic what we do in Core when it comes to code style.